client/src/components/application/Transactions.vue

```diff
@@ -60,9 +60,20 @@
             </v-layout>
           </div>
         </template>
-
-        <v-layout id="transaction-title-bar" text-xs-center>
-          <v-flex class="py-2 fs-18 fw-700">TRANSACTION HISTORY</v-flex>
+        <v-layout id="transaction-title-bar">
+          <v-flex class="ml-4 pl-3 pt-1 fs-16 fw-700">TRANSACTION HISTORY</v-flex>
+          <v-flex class="mr-4 pr-3 justify-end">
+            <v-checkbox
+              off-icon="mdi-checkbox-blank"
```

Comment:
I had to pull in a CDN and apply this icon as a blank filled checkbox wasn't available in the current import.

Reply:
Would it make sense to host the icon file locally?

Would it be possible to just have the 1 icon as a local file?

Also please create an icon to fully port this project to MDI instead of both MDI and MI.

Reply:
We could try and import just the 1 icon.

To do the latter is going to be re-working ALL the icons in the project to accommodate 1 icon?

Reply:
If you can import just the 1 icon then the existing code is ok (except when UI team wants us to have common icons with other projects, but that's a future bridge to cross).

But atm this imports 2 icon libraries :/

Reply:
Typo above: not "please create icon".

I meant "please create ticket".

Reply:
Ok will do, definitely agree. 
I was just concerned about making waves in a staff-only UI but i think you are the expert on this matter. A ticket would be prudent i think.

Reply:
Ticket: https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/8316